### ISSUE TITLE
Improve Performance

### ISSUE THREAD

**Issue by imchell** [postcss/postcss.org#277]
`section.doc` is an extemely large DOM. It affects rendering time and readability. Maybe we can truncate it by section and load it into different pages just like many other documents do?

**Comment by ai** [comment]
We will not have full text search in this case

**Comment by imchell** [comment]
Would some libs like Fuse.js be helpful?

**Comment by ai** [comment]
But Fuse.js is not simple in-browser search which everone knows

**Comment by imchell** [comment]
Yes. But if the doc is going to be truncated and manipulated by JS, it's reasonable that the doc data can be used to implemented search function powered by Fuse.js.
